src/components/Pressable/GenericPressable/BaseGenericPressable.js

```diff
@@ -0,0 +1,158 @@
+import React, {
+    useCallback,
+    useEffect,
+    useMemo,
+    forwardRef,
+} from 'react';
+import {Pressable} from 'react-native';
+import _ from 'lodash';
```

Comment:
Normally we use `underscore` instead of `lodash`. Not sure that it really matters 🤷🏼